tests/test_lib_thread_names.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+from juniorguru_chick.lib.thread_names import name_thread
+
+@pytest.mark.parametrize('message, expected_name', [("[eslint, nextjs]", "eslint, nextjs"), ("[eslint,nextjs]", "eslint,nextjs"), ("[ Java ]", "Java"), ("[:css: CSS]", "CSS"), ("[eslint,nextjs, :css: CSS, ruby]", "eslint, nextjs, CSS, ruby")])
```

Comment:
You probably want to have one per line - not per code style, but per having a good overview as a human.